core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -4165,14 +4165,6 @@ class ReplicaManagerTest {
       val response = replicaManager.becomeLeaderOrFollower(0, leaderAndIsrRequest(0), (_, _) => ())
       assertEquals(Errors.NONE, response.partitionErrors(topicNames).get(topicPartition))
       assertFalse(replicaManager.localLog(topicPartition).isEmpty)
-      val id = topicIds.get(topicPartition.topic())
```

Comment:
Looking at this again, does this test actually make sense to keep? It's supposed to test the metadata file get created but we've now removed these assertions.

Reply:
Yeah, I also wasn't sure if removing the assertions make sense here. I was planning to take a closer look. If the assertions are not useful, then we should check if the test is useful.

Reply:
> Looking at this again, does this test actually make sense to keep? It's supposed to test the metadata file get created but we've now removed these assertions.

If we're removing the created file assertion, we should also remove this test. This make sense to me.